src/test/java/com/wikia/webdriver/testcases/mobilewikitests/HTMLTitleTests.java

```diff
@@ -36,62 +36,62 @@
   private String[][] testCases = {
       {
           "sktest123",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "sktest123" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)